docs/installation.rst

```diff
@@ -19,17 +19,21 @@ On ``linux``, you can install the Python 3 version into ``$HOME/miniconda3`` wit
 
    $ wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
    $ bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda3
-   $ export PATH="$HOME/miniconda3/bin:$PATH"
+   $ source ~/miniconda3/etc/profile.d/conda.sh
+   $ conda activate base
```

Comment:
Isn't `conda activate base` superfluous?

Reply:
The preferred way of interacting with conda has changed, and they're having trouble getting the community to adopt it.

https://github.com/conda/conda/blob/master/CHANGELOG.md#440-2017-12-20

Basically, we shouldn't be adding `*conda/bin` directories to our `PATH` any more. Instead, they recommend sourcing the `profile.d/conda.sh` script. Under this strategy, `$ python` hits the system python _until_ we run `conda activate`. 

I'm ambivalent about the `base` part. I think I err on the side of liking it, because it shows new users that you could have many named conda environments, but I'd be happy to remove it if we don't want it.

Reply:
Thanks for the heads-up! I'll update our omnia docs too.